python-spec/src/somacore/experiment.py

```diff
@@ -66,15 +66,15 @@ class Experiment(
     spatial = _mixin.item[_SceneColl]()  # TODO: Discuss the name of this element.
     """A collection of named spatial scenes."""
 
-    # TODO: Discuss the name of this element.
-    # TODO: Discuss the best location for this element. Should it be moved
-    # inside the spatial collection?
-    # TODO: Use DF or new array type for int/string indexing?
-    obs_spatial = _mixin.item[_DF]()
-    """A dataframe mapping obs_id to scene names.
-
-    This provides a join table for the obs ``soma_joinid`` and the ``scene_ids``
-    used in the ``spatial`` collection.
+    obs_scene = _mixin.item[_DF]()
```

Comment:
We can't use a `SparseNDArray` in this case without creating an integer index for the scenes. Because we want to (for now) directly access the `scene_id` (a string), we need the greater flexibility of the DataFrame.

Reply:
Thank you!